src/ops_traits.rs

```diff
@@ -0,0 +1,229 @@
+#[cfg(feature = "ops-traits")]
```

Comment:
should you not just use it once for the entire file? why copy paste this line everywhere